packages/core/bootstrap/src/lib/server.ts

```diff
@@ -65,14 +64,16 @@ export const initHandler =
     })
 
     app.get(join(baseUrl, 'health'), async (_, res) => {
-      if (cacheOptions.enabled && cacheOptions.cacheImplOptions.type === 'redis') {
-        logger.debug('Checking if redis connection initialized')
-        const cache = context.cache.instance as redis.RedisCache
-        if (!cache.client.connected) {
-          res.status(500).send({ message: 'Redis not connected', version })
-          return
-        }
-      }
+      // TODO add health check for Redis connection (the following is deprecated)
```

Comment:
Do we have a Shortcut story we can link to here?

Reply:
e499557